.rubocop_todo.yml

```diff
@@ -10,3 +10,14 @@
 RSpec/AnyInstance:
   Exclude:
     - 'spec/channels/application_cable/connection_spec.rb'
+
+# Offense count: 1
+# Cop supports --auto-correct.
+# Configuration parameters: SuggestedStrictness, Include.
+# Include: **/*.rb, **/*.rbi, **/*.rake, **/*.ru
+Sorbet/FalseSigil:
+  Exclude:
+    - 'bin/**/*'
+    - 'db/**/*.rb'
+    - 'script/**/*'
+    - 'config/initializers/dry_types.rb'
```

Comment:
It didn't seem worth the time to work out how to get dry-types and sorbet to live together harmoniously.